routes/follow.ts

```diff
@@ -48,13 +57,36 @@ function follow(query: { follower: string; following: string; }, isFollower = tr
           .update(
             {_id: updatingUserId},
             {$pull: updatingField})
-          .then(() => `You have started following ${user.username}`);
+          .then(() => ({isFollowed: false, message: `You have stopped following ${user.username}`}));
       }
 
       return users
         .update(
           {_id: updatingUserId},
           {$push: updatingField})
-        .then(() => `You have stopped following ${user.username}`);
+        .then(() => ({isFollowed: true, message: `You have started following ${user.username}`}));
+    });
+}
+
+function checkFollowed(req: Request, res: Response): void | undefined {
+  const body = req.body;
+
+  if (!body.follower || !body.following) {
+    const status = 500;
+    res.status(status).send({message: 'follower or followed are absent'});
+
+    return undefined;
+  }
+
+  users.findOne({_id: body.follower})
+    .lean(true)
+    .exec()
+    .then((user: User) => {
```

Comment:
user =>

Reply:
If I do it I have TSLint error `TS2339: Property 'statistics' does not exist on type 'Object'`